### PR TITLE
[SMALLFIX] Replace with Map.computeIfAbsent

### PR DESCRIPTION
**Pull request by Ctto**
…rker/src/main/java/alluxio/worker/block/DefaultBlockStoreMeta.java#getDirectoryPathsOnTiers

### CODE REVIEW

File: core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockStoreMeta.java [hunk 1]
```diff
@@ -84,9 +84,7 @@ public long getCapacityBytes() {
     Map<String, List<String>> pathsOnTiers = new HashMap<>();
     for (Pair<String, String> tierPath : mCapacityBytesOnDirs.keySet()) {
       String tier = tierPath.getFirst();
-      if (pathsOnTiers.get(tier) == null) {
-        pathsOnTiers.put(tier, new ArrayList<String>());
-      }
+      pathsOnTiers.computeIfAbsent(tier, k -> new ArrayList<>());
```
Comment (cheng-chang) [inline core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockStoreMeta.java]: why not `xxx.computeIfAbsent(xxx).add(xxx)`, no reason to `get(tier)` again in the line below.

Comment (Ctto) [reply core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockStoreMeta.java]: Thanks for your advice.

**Comment by Ctto** [comment]
alluxio-bot, check this please

**Comment by Ctto** [comment]
@calvinjia PTAL. Thanks.

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by calvinjia** [comment]
@Ctto Thanks for improving the code quality!

**Pull request merged**